SimulationAgent/appsettings.ini

```diff
@@ -5,12 +5,37 @@ iothub_connstring = "${PCS_IOTHUB_CONNSTRING}"
 device_models_folder = ./data/devicemodels/
 device_models_scripts_folder = ./data/devicemodels/scripts/
 
+[DeviceSimulationService:RateLimits]
+# S3:  5000/min/unit (= 83.3/sec/unit)
+# S2:   100/min/unit (= 1.67/sec/unit)
+# S1:   100/min/unit (= 1.67/sec/unit)
+# F1:   100/min      (= 1.67/sec)
+registry_operations_per_minute = 100
+# S3:  50/sec/unit
+# S2:   1/sec/unit  (10/sec minimum)
+# S1:  10/sec
+# F1:  10/sec
+twin_reads_per_second = 10
+# S3:  50/sec/unit
+# S2:   1/sec/unit  (10/sec minimum)
+# S1:  10/sec
+# F1:  10/sec
+twin_writes_per_second = 10
+# S3:  6000/sec/unit
+# S2:   120/sec/unit
+# S1:    12/sec/unit (100/sec minimum)
```

Comment:
nit, higher of 12/sec/unit or 100/sec (not 100/sec min)

https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-quotas-throttling

Reply:
changed to "higher of 100/sec and 12/sec/unit"